core/src/datayoga_core/blocks/map/tests/test_map.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+from datayoga_core import result
+from datayoga_core.blocks.map.block import Block
+
+
+@pytest.mark.asyncio
+async def test_map_expression_jmspath():
+    block = Block(properties={"language": "jmespath",
+                              "expression": """{
+                        "new_field": `hello`
+                   }"""})
+    block.init()
+    assert await block.run([{"fname": "john", "lname": "doe"}]) == ([
+        {"new_field": "hello"}], [result.SUCCESS]
+    )
+
+
+@pytest.mark.asyncio
+async def test_map_field_multiple_expressions_jmspath():
+    block = Block(properties={"language": "jmespath",
+                              "expression": """{
+                            "new_field": `hello`, "name" : fname
+                    }"""})
+    block.init()
+    assert await block.run([{"fname": "john", "lname": "doe"}]) == ([
+        {"new_field": "hello", "name": "john"}], [result.SUCCESS]
+    )
+
+
+@pytest.mark.asyncio
+async def test_map_field_nested_expression_jmspath():
+    block = Block(properties={"language": "jmespath",
+                              "expression": """{
+                            "new_field": `hello`, "name" : details.fname
+                    }"""})
+    block.init()
+    assert await block.run([{"details": {"fname": "john", "lname": "doe"}}]) == ([
+        {"new_field": "hello", "name": "john"}], [result.SUCCESS]
+    )
+
+
+@pytest.mark.asyncio
+async def test_map_field_double_nested_expression_jmspath():
+    block = Block(properties={"language": "jmespath",
```

Comment:
since the only argument the constructor takes is the `properties`, I think it's redundant to pass it by its name.